### PR TITLE
DBZ-2552 Code cleanup & refactorings

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-2552

@gunnarmorling This PR is almost done and I tried to break each tiny refactor into a separate commit to make the review easier.  We can always squash all this work after-the-fact if you'd prefer one commit when we're done.

A vast majority of these commits are taking code from `LogMinerHelper` and `SqlUtils` and putting it in a more appropriate place which is inside the `LogMinerStreamingChangeEventSource` class.

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Hey @Naros, given this is largely about refactoring, is it something we can keep for 1.7 Alpha1? Or is their any functional change that should go into 1.6 Final? Given we're at the CR stage, we should limit the changes that go in at this point.

**Comment by Naros** [comment]
Hi @gunnarmorling, I'm fine letting this wait until 1.7 since 1.6 Final is right around the corner.

**Comment by Naros** [comment]
@gunnarmorling re-based against master once again.

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/SqlUtilsTest.java [hunk 1]
```diff
@@ -7,8 +7,6 @@
```
Comment (jpechane) [inline debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/SqlUtilsTest.java:7]: @Naros Should not be there a replacement for delete test segments? Maybe the flush strategies needs their own tests?

Comment (Naros) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/SqlUtilsTest.java:7]: I'll take a look at and see if it's possible but a number of these were removed simply due to dead-code and no other reason.

**Comment by jpechane** [comment]
@Naros ok, merging thanks!

**Pull request merged**